### ISSUE TITLE
Identity box changes

### ISSUE THREAD

**Issue by NomDeMorte** [overdodactyl/ShadowFox#279]
Latest nightly has made significant change within thte URL bar, including identity-box. Colour for padlock icon is incorrect:

**
![image](https://user-images.githubusercontent.com/45681208/61593253-89a91000-ac20-11e9-9d9c-110cc921fd11.png)
**

**Comment by loopy750** [comment]
Apart from the padlock, I'm not seeing anything else that looks out of place.

A manual fix: line 1218 of userChrome.css, replace

`#connection-icon,`

with

`#connection-icon, #identity-icon,`

**Comment by NomDeMorte** [comment]
Yep colour-wise and in the context of shadowfox currently, only that padlock icon I mentioned is an issue. 

I only mention the other changes because there have actually been quite a few over the past nightlies and one could reasonably anticipate that this padlock icon may not be the only one that will change. In other words: They're re-doing the entire url-bar so maybe this is just the first thing they break for SF.

**Comment by NomDeMorte** [comment]
As I suspected - the tracking protection icon has now changed colour also.

**Comment by NomDeMorte** [comment]
The identity popup has changed today.

Old code:
`#identity-popup-permissions-content {
  background-image: url(data:image/svg+xml;base64........
}
.identity-popup-security-content {
  background-image: url(data:image/svg+xml;base64,..........
}`

Causes these images to be repeated covering the whole background. Stopping the repeat works, but then the images are misaligned. The following looks OK as a quick workaround but it could use some attention.

#identity-popup-permissions-content {
  background-image: url(data:image/svg+xml;base64...................
  background-repeat: no-repeat !important;
  background-position: 2px 12px !important;
}
.identity-popup-security-content {
  background-image: url(data:image/svg+xml;base64,.................
  background-repeat: no-repeat !important;
  background-position: 2px 12px !important;
}


By the way, when changing this I had some trouble which led me to notice something which is syntactically allowable but somewhat inconsistent. There are quite a few lines which are not closed with a semicolon but are followed with a closed bracket. Of course, the semicolon is redundant with the bracket following it, but the lines with the base64 encoded images show exactly why it's nice to have that redundancy - because someone editing the code and placing a line between the unterminated line without the semicolon, and the bracket, cannot see that the line is not terminated with a semicolon, because they cannot see the end of the line:

![image](https://user-images.githubusercontent.com/45681208/62755865-ffcec300-bab8-11e9-81ad-e35d3038c459.png)

Up to you, but me personally, I'd be doing a quick search and replace on all those !important without the semicolons after them....

**Comment by NomDeMorte** [comment]
The identity box now uses the ::before pseudo-element to apply a badge image to the padlock icon.

There is also some new spacing and a new separator added.

The changes just keep coming.........

**Comment by krystian3w** [comment]
```css
#identity-popup-permissions-content, .identity-popup-security-content {
  background-repeat: no-repeat !important;
  background-position: 2px 12px !important;
}
```

paste on end in `userChrome.css`.
